spec/requests/broadcast_announcements_spec.rb

```diff
@@ -0,0 +1,20 @@
+require "rails_helper"
+
+RSpec.describe "BroadcastAnnouncements", type: :request do
+  before(:all) do
+    @test_a = create(:broadcast_announcement, link: "http://google.com")
+  end
+
+  it "creates an announcement" do
+    expect(@test_a).to be_valid
+  end
+
+  it "can find an announcement" do
+    expect(BroadcastAnnouncement.find_by(message: "test")).to eq(@test_a)
+  end
+
+  it "deletes an announcement" do
+    @test_a.destroy
+    expect(BroadcastAnnouncement.count).to eq(0)
+  end
+end
```

Comment:
There need to be request specs as well as a check on the dashboard spec if an announcement exists that it shows up.